### PR TITLE
#33: added dependencies and fixed javaDoc warnings

### PR DESCRIPTION
**Pull request by AnastasiiaSergienko**

### CODE REVIEW

File: src/main/java/com/exasol/adapter/sql/AggregateFunction.java [hunk 1]
```diff
@@ -27,9 +27,11 @@
 
     /**
      * True if the function is simple, i.e. is handled by {@link SqlFunctionAggregate}, and false if it has it's own implementation.
+     *
+     * @return if the function is simple
```
Comment (redcatbear) [inline src/main/java/com/exasol/adapter/sql/AggregateFunction.java]: @return <code>true</code> if ...

File: src/main/java/com/exasol/adapter/sql/ScalarFunction.java [hunk 2]
```diff
@@ -188,9 +188,11 @@
 
     /**
      * True if the function is simple, i.e. is handled by {@link SqlFunctionScalar}, and false if it has it's own implementation.
+     *
+     * @return if the function is simple
```
Comment (redcatbear) [inline src/main/java/com/exasol/adapter/sql/ScalarFunction.java]: dto.

File: src/main/java/com/exasol/adapter/sql/AggregateFunction.java (hunk 1 above)
Comment (AnastasiiaSergienko) [reply src/main/java/com/exasol/adapter/sql/AggregateFunction.java]: done

File: src/main/java/com/exasol/adapter/sql/ScalarFunction.java (hunk 2 above)
Comment (AnastasiiaSergienko) [reply src/main/java/com/exasol/adapter/sql/ScalarFunction.java]: done

**Pull request merged**